tfhe/Cargo.toml

```diff
@@ -324,3 +330,10 @@ required-features = ["integer", "internal-keycache"]
 
 [lib]
 crate-type = ["lib", "staticlib", "cdylib"]
+
+[lints.rust]
+unexpected_cfgs = { level = "warn", check-cfg = [
+    'cfg(__profiling)',
```

Comment:
Not sure about this one
There's a feature `__profiling` which may be misused
@sarah-ek do you know if the pattern `#[cfg_attr(__profiling, inline(never))]` works as expected?

Reply:
no. it should be feature = "__profiling"
nice catch